tests/functional/object/mcg/test_namespace_store.py

```diff
@@ -0,0 +1,102 @@
+import logging
+
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+from ocs_ci.helpers.helpers import create_unique_resource_name
+from ocs_ci.ocs.bucket_utils import (
+    write_random_objects_in_pod,
+    write_individual_s3_objects,
+)
+from ocs_ci.ocs.resources.objectbucket import OBC
+from ocs_ci.ocs.resources.cloud_manager import CloudManager
+from ocs_ci.ocs.resources.namespacestore import (
+    cli_create_namespacestore,
+    NamespaceStore,
+)
+from ocs_ci.framework.pytest_customization.marks import polarion_id, mcg, red_squad
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.ocs import OCS
+
+logger = logging.getLogger(__name__)
+
+
```

Comment:
Do we need to include the tier marker as well?

Reply:
yes will add that